core/src/main/java/org/wildfly/extras/creaper/core/online/OnlineOptions.java

```diff
@@ -356,9 +356,6 @@ ModelControllerClient createModelControllerClient() throws IOException {
         if (wrappedModelControllerClient != null) {
             return wrappedModelControllerClient;
         }
-        if (protocol == ManagementProtocol.HTTP) {
-            return new HttpModelControllerClient(host, port, username, password, connectionTimeout);
-        }
```

Comment:
I actually liked this, the rest of this method is quite involved and the less we touch it, the better. I just wanted to call that one method :-)